### DISCUSSION TITLE
Save As function?

### DISCUSSION THREAD

**Discussion by Jezzerisin** [mifi/lossless-cut#1599] [Q&A] [answered]
Dear Friends,
I love Lossless Cut!
I used to be able to edit the file name after pressing the Export Button, but now it just saves straight over an old file.
I guess I'm doing something wrong! Can you help..

Am I also correct in thinking that I have to run Lossless Cut from my desktop, and that it doesn't install into windows (i.e that it's more like and App or Executable than in installation?) Happy if that's the case, but wanted to check I'm not missing anything!

I use Windows 11 (for my sins..) and a MS Surface (Microsoft own's my ass...)

Jeremy

**Comment by hrxn** [comment]
Depends on your definition of "install"... it does save its application preferences inside a directory in `C:\Users\<USERNAME>\AppData\Roaming`

It used to, at least..

**Reply by mifi** [reply, marked as answer]
Hi. maybe you need to enable the export confirmation dialog - this button:
<img width="70" alt="Screenshot 2023-05-14 at 23 21 23" src="https://github.com/mifi/lossless-cut/assets/402547/1fda35e5-0eab-45e8-bc78-5066971cb6f4">

**Reply by Jezzerisin** [reply]
Mikael,
You are a Bona Fide Genius!
I guess I must have touched that On/Off switch accidently!
It's not 100% clear what the function of that button is - looks more like a calendar icon, so I didn't think to try it!

Just a tiny adjustment, but saves lots of frustration!
